c/public/lib/source/cffwrite/cffwrite_t2cstr.c

```diff
@@ -361,16 +364,34 @@ static void saveopDirect(cstrCtx h, int op) {
         h->stack.cnt = 0;
     }
 
+    h->flushed_cnt = 0;
     cstr_saveop(h, op);
 
     h->pendop = tx_noop; /* Clear pending op */
 }
 
+/* Returns 1 if argcnt maybe blended args may overflow the operand stack */
```

Comment:
This comment is not clear to me. Did you mean "...argcnt **plus** blended args **will** overflow..."?

Reply:
correct